CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README.md) - Common resources for existing developers
```

Comment:
```suggestion
- [Contributor Cheat Sheet](https://k8s.dev/cheatsheet) - Common resources for existing developers
```
Thanks, just have 1 suggestion
We are trying to direct more folk to the k8s.dev website as its more easily searchable than directly in github.

Also as an FYI - for links that do directly route to a place in github, we do want to use our shortner. If the content wasn't on the site it would look like this:
https://git.k8s.io/community/contributors/guide/contributor-cheatsheet/README.md

Reply:
thank you for suggestions @mrbobbytables.